### PR TITLE
Per 9349 storage redeemed does not show in storage bar

### PR DESCRIPTION
**Pull request by crisnicandrei**
I have updated the components to set the account with the new values after the api calls. I have also added tests to the redeem storage component. @meisekimiu I also wanted to add some tests to the pledge component, but I was unsuccessful, mainly because I was not able to mock the Stripe api calls. Do you have any suggestions for it? Because I would like to add some as well. Thanks!

### CODE REVIEW

**Comment by meisekimiu** [comment]
You do not really need to mock Stripe API calls, since Stripe's API is out of our control. We really only need to test the logic in the components themselves and not external libraries like Stripe.

**Comment by crisnicandrei** [comment]
@meisekimiu I see. How could i test the submit without stripe since the call is made inside the submit?

Or what do you suggest i should test?

**Comment by crisnicandrei** [comment]
@meisekimiu more tests added

**Review by meisekimiu — APPROVED** [review]
Thanks for writing all the tests! This seems to be functioning properly! I have a few small comments on some adjustments that I think should be made, but this mostly looks good.

File: src/app/pledge/components/new-pledge/new-pledge.component.ts [hunk 1]
```diff
@@ -288,6 +298,10 @@ export class NewPledgeComponent implements OnInit, AfterViewInit, OnDestroy {
     }
   }
 
+  private getSizeInBytes(amount: number): number {
+    return Math.floor(amount / 10) * 1073741824;
```
[the remark this answers is not in the document]
Comment (crisnicandrei) [reply src/app/pledge/components/new-pledge/new-pledge.component.ts:302]: @k8lyn6 can you please let me know which ones are we using?

**Comment by crisnicandrei** [comment]
@meisekimiu I did the suggested changes, and after moving the function inside the account service, the test fails. I have mocked it inside the test file as I did with the other mocks, but it does not seem to get called. Could you please take a look at the code and let me know if you can notice what I am doing wrong? It was working before so I am very confused.

File: src/app/pledge/components/new-pledge/new-pledge.component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (cecilia-donnelly) [reply src/app/pledge/components/new-pledge/new-pledge.component.ts]: We use GiB. @k8lyn6, if you're interested: https://massive.io/file-transfer/gb-vs-gib-whats-the-difference

**Review by k8lyn6 — APPROVED** [review]
Looks good!

**Pull request merged**